src/js/helpers.js

```diff
@@ -436,16 +436,16 @@ export const standardizePhases = phases =>
   });
 
 export const getDebConfigurationCode = (ipAddress, isHosted, isEnterprise, token, packageVersion, deviceType = 'generic-armv6') => {
-  let connectionInstructions = `  --demo ${ipAddress ? `--server-ip ${ipAddress}` : ''}`;
+  let connectionInstructions = `  --quiet --demo ${ipAddress ? `--server-ip ${ipAddress}` : ''}`;
   if (isEnterprise || isHosted) {
     const enterpriseSettings = `  --tenant-token $TENANT_TOKEN \\
   --retry-poll 30 \\
   --update-poll 5 \\
   --inventory-poll 5`;
-    connectionInstructions = `${ipAddress ? `  --server-url ${ipAddress}` : ' '} --server-cert="" \\
-${enterpriseSettings}`;
+    connectionInstructions = `--quiet ${ipAddress ? `  --server-url ${ipAddress}` : ' '} \\
+    --server-cert="" {enterpriseSettings}`;
     if (isHosted) {
-      connectionInstructions = `  --mender-professional \\
+      connectionInstructions = `  --quiet --hosted-mender \\
```

Comment:
I don't think you need `--quiet` here, as it will be adding it twice

Reply:
@mzedel What about this ^ ? Am I correct?

Reply:
no the `connectionInstructions` will be overwritten - as the (potentially set) `server-url` and `server-cert` would cause problems otherwise...
(initially I thought the "adding it twice" was referring to the internal implementation of the mender setup)

Reply:
Ok, thanks. My bad.